src/components/SitewideHeader/SitewideHeader.js

```diff
@@ -86,39 +88,45 @@ const SitewideHeader = ({
   ctaTo = '/request-demo/',
   ctaText = 'Get a Demo',
 }) => (
-  <Popover as={React.Fragment}>
-    <>
-      <header
-        className={classnames(
-          ` max-w-${maxWidth} mx-auto px-2 py-6 sm:px-6 xl:py-8 flex justify-between items-center  md:justify-start md:space-x-10`,
-          {
-            'border-b-2 border-gray-100': borderBottom,
-          }
-        )}
-      >
-        <div className="flex justify-start lg:w-0 lg:flex-1">
-          <Logo />
-        </div>
+  <>
+    <TopBanner>
+      <PragmaticEngineerNewsletter />
+    </TopBanner>
 
-        <div className="-mr-2 -my-2 md:hidden">
-          <OpenMenuButton />
-        </div>
+    <Popover as={React.Fragment}>
+      <>
+        <header
```

Comment:
These are not really changes. I just increased indentation and wrapped in a Fragment.